Update Web OTP article (#4094)
GoogleChrome/web.dev: src/site/content/en/blog/web-otp/index.md

1. Send your phone the following SMS text message from the another phone.

```text
Your OTP is: 123456.

@web-otp.glitch.me #12345
```

Did you receive the SMS and see the prompt to enter the code to the input area?
That is how the Web OTP API works for users.

> {% Aside 'caution' %}

Reviewer: ```suggestion
{% Aside 'gotcha' %}
```
Our gotcha callout is more appropriate now

> The dialog doesn't appear for you? Check out [the FAQ](#no-dialog).

Reviewer 2: Why not just explain it here?

Reviewer: ```suggestion
If the dialog doesn't appear for you, check out [the FAQ](#no-dialog).
```

Reviewer: I'm OK with how Eiji has handled it. Per https://kayce.basqu.es/blog/FAQs/ (which was inspired by you, Joe), the main thing is that Eiji has mentioned the topic in a contextually relevant place. I'm OK with us having an FAQ, so long as that isn't the only source of the information.

where `get()` was called. Finally, the message must adhere to the following
formatting:

> * The message begins with (optional) human-readable text that contains a 3-10

Reviewer 2: ```suggestion
* The message begins with (optional) human-readable text that contains a three to ten
```

## FAQ

> ### The dialog doesn't appear though I'm sending a properly formatted message. What's going wrong? {: #no-dialog}

Reviewer 2: Think about the order that you have implied by this. You didn't give these instruction above. Now the reader who thinks they got everything has to figure out why their code doesn't work. Then they find this and thinks, "Why didn't they provide this information to begin with?"

Author: I wanted to keep this in FAQ but wanted to avoid duplicate notes earlier. Yes, I know it should be noted earlier as well. Do you think it's ok to write the same thing twice in this article?

Reviewer 2: No. What I'm saying is, if the article covers this in a logical place to begin with, why would it need to be a frequently asked question?

Author: Because I'm assuming this will be asked frequently. It's easier for readers to look for FAQ than read from the beginning to find the information. I would keep it at FAQ.

Reviewer: I'm OK with the current organization. Eiji has mentioned the topic in a contextually relevant place within the main flow of the document. If this content was only in the FAQ, and nowhere else, I would side with Joe. FAQs are a very common pattern and it's plausible that some people find them useful. The main thing is to ensure that we don't get lazy and only use FAQs to present information. See https://kayce.basqu.es/blog/FAQs/
